### PR TITLE
Parallel Package Documentation & misc 

### LINKED ISSUE #468: Add documentation for the parallel subpackage
**Is your feature request related to a problem? Please describe.**
It was noted in #466 that there is no documentation for the functions in the `plantcv.parallel` subpackage.

**Describe the solution you'd like**
We need to add a `Package plantcv.parallel` section to the documentation with a page describing the use of each function.

### PR DESCRIPTION
**Pull request by HaleySchuhl**
**Describe your changes**
Add documentation pages for each of the functions in the PlantCV Parallel package. 

**Type of update**
* Update to documentation

**Associated issues**
Closes #468 
This PR is not easy to merge until #464 is merged into master 

**Additional context**
Also added misc things that needed updating (renamed parameter in `pcv.apply_mask` from `rgb_img` to `img` since it can be any type of image). 

**THIS NEEDS TO BE MERGED AFTER THE HYPERSPECTRAL PR**
Whoops.. It's going to be too messy to try to do this PR first.

### CODE REVIEW

File: docs/jupyter.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ First, if PlantCV is installed in the global Python search path, you can
 import the PlantCV library like normal:
 
 ```python
-import plantcv
```
Comment (HaleySchuhl) [inline docs/jupyter.md]: Is this true?

File: tests/hyperspectral_data/darkReference2.hdr [hunk 2]
```diff
@@ -5,7 +5,7 @@ lines = 1
 bands = 978
 header offset = 0
 file type = ENVI Standard
-data type = 12
+data type = 4
```
Comment (nfahlgren) [inline tests/hyperspectral_data/darkReference2.hdr]: Should this file be updated in this PR?

File: tests/hyperspectral_data/darkReference3.hdr [hunk 3]
```diff
@@ -5,7 +5,7 @@ lines = 1
 bands = 978
 header offset = 0
 file type = ENVI Standard
-data type = 12
+data type = 4
```
Comment (nfahlgren) [inline tests/hyperspectral_data/darkReference3.hdr]: This file also?

File: docs/jupyter.md (hunk 1 above)
Comment (nfahlgren) [reply docs/jupyter.md]: Nope, should be revised to `from plantcv import plantcv` as you have it

**Pull request merged**